OpenTabletDriver.Configurations/Parsers/Veikk/VeikkV1ReportParser.cs

```diff
@@ -2,13 +2,16 @@
 
 namespace OpenTabletDriver.Configurations.Parsers.Veikk
 {
-    public class VeikkV2ReportParser : IReportParser<IDeviceReport>
+    public class VeikkV1ReportParser : IReportParser<IDeviceReport>
     {
         public IDeviceReport Parse(byte[] report)
         {
+            if (report[0] == 0x03)
+                return new VeikkAuxV1Report(report);
+
             return (report[1], report[2]) switch
             {
-                (0x41, byte x) when (x & 0xF0) == 0xA0 => new TabletReport(report[1..]),
+                (0x41, _ ) when (report[2] & 0xF0) == 0xA0 => new TabletReport(report[1..]),
```

Comment:
Also revert this change. This is done as an easy micro-optimization that the JIT compiler may miss.
```suggestion
                (0x41, byte x) when (x & 0xF0) == 0xA0 => new TabletReport(report[1..]),
```